Content.Shared/Slippery/SlipperySystem.cs

```diff
@@ -63,6 +66,13 @@ private void TrySlip(EntityUid uid, SlipperyComponent component, EntityUid other
         if (HasComp<KnockedDownComponent>(other))
             return;
 
+        if (HasComp<ThrownItemComponent>(uid))
+        {
+            // This popup is here so people don't just think the game broke, it can probably be removed later
+            _popup.PopupEntity(Loc.GetString("thrown-slippery-missed"), other);
+            return;
+        }
+
         var attemptEv = new SlipAttemptEvent();
         RaiseLocalEvent(other, attemptEv);
         if (attemptEv.Cancelled)
```

Comment:
Use the attempt it's there, also popupclient

Reply:
Can you please elaborate on what you want the attempt to be used for, or what the issue is?

Client will only show to the specified entity, won't it? I think it would be better -at least initially- for witnesses to see the MISS text too, so for example Sec who sees the clown fail to slip a passenger doesn't incorrectly determine passenger is wearing noslips

Reply:
move the code to be a subscription to SlipAttemptEvent

Reply:
Oh, hmm. A ran into a problem with this. SlipAttempt is raised on the mob being slipped. The thrown item component is on the soap. There is a SlipEvent raised on the soap, but that is no longer cancellable. Should I make a new event for this? Or just leave it as is

Reply:
Either raise it on that as well or make a new event.

Reply:
I added the check on SlipEvent and made it cancellable. 
A Breaking Change warning has been added to the PR